build_runner/test/generate/serve_test.dart

```diff
@@ -61,36 +62,36 @@ a:file://fake/pkg/path
       await wait(100);
 
       var request = Request('GET', Uri.parse('http://localhost:8000/a.txt'));
-      // ignore: unawaited_futures
-      (webHandler(request) as Future<Response>).then((Response response) {
+      unawaited((webHandler(request) as Future<Response>)
+          .then(expectAsync1((Response response) {
```

Comment:
I also added some expectAsync since they looked missing - we want to ensure these expects actually run.